### PR TITLE
Forgotten Thief Quest Fix, and entered the Mausoleum of Karail Fix, and () Description Fixes

### PR DESCRIPTION
**Pull request by ttgmichael**
Finally renames "and entered the Mausoleum of Karail"

to 

"Carlyle Mausoleum"

xD

screenshot_20151215_00107

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -641,23 +641,23 @@ QUEST_LV_0100_20150317_000640	Coben
...
 QUEST_LV_0100_20150317_000642	$He was a really fantastic person.{nl}I can't imagine myself being like him.
 QUEST_LV_0100_20150317_000643	$Something is between the pages of this diary.{nl}A blueprint.. and oh, my brother had recorded a few things here.
-QUEST_LV_0100_20150317_000644	$I came here to look for my brother, but I am not as brave as he was.{nl}I want you to take a look at his gravestone's writing on my behalf.
-QUEST_LV_0100_20150317_000645	$Yes. My brother was a thief.{nl}My brother used to come to our shop once in a few years and tell us stories about his adventures.
-QUEST_LV_0100_20150317_000646	$So he challenged the Archmage Tower?{nl}That's my brother! He could certainly do that.{nl}
+QUEST_LV_0100_20150317_000644	$I came here to look for my brother, but I am not as brave as he is.{nl}I want you to take a look at his monuments' writing on my behalf.
+QUEST_LV_0100_20150317_000645	$Yes. My brother is a thief.{nl}My brother used to come to our shop once in a few years and tell us stories about his adventures.
+QUEST_LV_0100_20150317_000646	$So he challenged the great Mage Tower?{nl}That's my brother! He could certainly do that.{nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:646]: Needs to be reverted. Archmage Tower. (which is also an instanced dungeon)

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -7757,7 +7757,7 @@ QUEST_LV_0100_20150730_007756	{memo X}To express my apology, I want to help the
 QUEST_LV_0100_20150730_007757	{memo X}Those gentiles! They used the name of the Goddess and did this...{nl}They should not insult the dead ones like this. I want chase them right away and make them pay for it...{nl}
 QUEST_LV_0100_20150730_007758	{memo X}But, as a grave guard, I think we should first stabilize Catacomb.{nl}Thanks. I realized that I also have to take care for the resurrected ones' rest as a grave guard.{nl}Since Bokor said he would help me, I will protect the ones that have remained.{nl}
 QUEST_LV_0100_20150730_007759	{memo X}The Tree Sect of the Truth crossed here
-QUEST_LV_0100_20150730_007760	and entered the Mausoleum of Karail
+QUEST_LV_0100_20150730_007760	$Carlyle Mausoleum
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:7760]: Carlyle's Mausoleum

File: QUEST_LV_0100.tsv (hunk 1 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:646]: QUEST_LV_0100_20150317_000646   is a dialog involving the brother that's trying to invade the Mage Tower (he died to Agailla Flurry). I'm pretty sure he isn't invading the instanced dungeon, which has no story driven for it.

The very next line that comes after this line is:
QUEST_LV_0100_20150317_000647   $The Mage Tower was a fortress that demons could not even try to approach.

So the speaker (the quest giver) is actually referring to the Mage Tower, not the Archmage Tower.

File: ETC.tsv [hunk 3]
```diff
@@ -14675,9 +14675,9 @@ ETC_20151001_014676	Lapeman
 ETC_20151001_014677	Member Alina Track
 ETC_20151001_014678	Chaser Werewolf
 ETC_20151001_014679	Kartas' Hunter Dullahan
-ETC_20151001_014680	Ashaq Basement Prison 1F
-ETC_20151001_014681	Ashaq Basement Prison 2F
-ETC_20151001_014682	Ashaq Basement Prison 3F
+ETC_20151001_014680	Axiac Vault Prison 1F
```
Comment (Sunaries) [inline ETC.tsv]: Ashaq Underground Prison 1F

File: ETC.tsv [hunk 4]
```diff
@@ -14675,9 +14675,9 @@ ETC_20151001_014676	Lapeman
...
+ETC_20151001_014681	Axiac Vault Prison 2F
```
Comment (Sunaries) [inline ETC.tsv]: Ashaq Underground Prison 2F

File: ETC.tsv [hunk 5]
```diff
@@ -14675,9 +14675,9 @@ ETC_20151001_014676	Lapeman
...
+ETC_20151001_014682	Axiac Vault Prison 3F
```
Comment (Sunaries) [inline ETC.tsv]: Ashaq Underground Prison 3F

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**